dashboard/templates/moj_template_custom/moj_base.html

```diff
@@ -142,11 +142,18 @@
         <div class="header-bar">
             {% block headerbar %}{% endblock %}
         </div>
+        <div class="nav-bar">
+            <ul>
+                <li class="product"><a href="/projects/1">Product Overview</a></li>
+                <li class="service"><a href="/services/1">Service Overview</a></li>
+                <li class="portfolio"><a href="/portfolio">Portfolio Summary</a></li>
+                <li class="admin"><a href="/admin">Admin</a></li>
```

Comment:
Maybe only show admin if user has access to it? I think there is one user type that doesn't have access